mlocal/frags/Makefile.stub

```diff
@@ -77,6 +81,18 @@ $(go_OBJ): $(BUILDDIR)/vendors-done $(libruntime) $(BUILDDIR)/config.h
 	@rm -f $(go_OBJ)
 	$(V)export BUILDDIR=`readlink -f $(BUILDDIR)` && cd $(SOURCEDIR)/src/pkg/buildcfg && go generate
 
+# bash_completion file
+$(bash_completion): $(go_OBJ)
+	@rm -f $(bash_completion)
+	@mkdir -p $(@D)
+	$(V) go run  $(SOURCEDIR)/etc/bash_completion.d/bash_completion.go $(bash_completion)
```

Comment:
Remove extra space between $(V) and go
and
change $(bash_completion) at the end of the line for : $@